src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/repository/ReviewScheduleRepository.kt

```diff
@@ -12,7 +12,7 @@ interface ReviewScheduleRepository : JpaRepository<ReviewScheduleEntity, UUID> {
 
   fun getAllByPrisonNumber(prisonNumber: String): List<ReviewScheduleEntity>
 
-  @Query("select rse from ReviewScheduleEntity rse where rse.prisonNumber = :prisonNumber and rse.scheduleStatus != 'COMPLETED'")
+  @Query("select rse from ReviewScheduleEntity rse where rse.prisonNumber = :prisonNumber and rse.scheduleStatus != 'COMPLETED' and rse.scheduleStatus != 'EXEMPT_PRISONER_RELEASE' and rse.scheduleStatus != 'EXEMPT_PRISONER_DEATH' and rse.scheduleStatus != 'EXEMPT_UNKNOWN'")
```

Comment:
I couldn't work out how to do an `in` type clause. EG: this gave compilation errors, so I went with this approach for now 🤷‍♂️ 
```
@Query("select rse from ReviewScheduleEntity rse where rse.prisonNumber = :prisonNumber and rse.scheduleStatus NOT IN ('COMPLETED', 'EXEMPT_PRISONER_RELEASE', 'EXEMPT_PRISONER_DEATH', 'EXEMPT_UNKNOWN')")
```
It didnt like how I was trying to express the values in the `in` clause because in the entity they are enums, not strings! Couldnt work out how to express them as enums 🤷‍♂️

Reply:
not 100% but I think this would work: 

@Query("SELECT rse FROM ReviewScheduleEntity rse WHERE rse.prisonNumber = :prisonNumber AND rse.scheduleStatus NOT IN (:excludedStatuses)")
List<ReviewScheduleEntity> findByPrisonNumberAndStatusNotIn(
    @Param("prisonNumber") String prisonNumber,
    @Param("excludedStatuses") List<ScheduleStatus> excludedStatuses
);

ie passing in enums rather than strings.

Reply:
Thanks for this 👍 
I think I've addressed it in this commit: https://github.com/ministryofjustice/hmpps-education-and-work-plan-api/pull/526/commits/98544876ecf341ce4060dfcfbd9bd898a8d41604

It involved a little bit of refactoring, and we can now do it without the `@Query` annotation and just let JPA do it's magic; but I think it's OK ....
Let me know what you think ...